dozer/cogs/teams.py

```diff
@@ -61,6 +64,49 @@ async def teamsfor(self, ctx, user: discord.Member = None):
     `{prefix}teamsfor member` - Returns all team associations with the mentioned user. Assumes caller if blank.
     """
 
+    @command()
+    @guild_only()
+    async def compcheck(self, ctx, event_type: str, event_key):
+        """Allows you to see people in the Discord server that are going to a certain competition."""
+        if event_type.lower() == "frc":
+            teams_raw = await TBA(ctx.bot).session.event_teams(event_key)
+            teams = [team.team_number for team in teams_raw]
+        elif event_type.lower() == "ftc":
+            teams_raw = json.loads(await TOA(ctx.bot).parser.req(f"/api/event/{event_key}/teams"))
+            teams = [team['team']['team_number'] for team in teams_raw]
```

Comment:
Creating a new `TBA`/`TOA` cog every time this command is called is likely going to cause bugs in the future. Dozer cogs assume they're only instantiated once. You can find the existing TBA/TOA cog with the [`Bot.get_cog`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Bot.get_cog) method.

Reply:
Oops, didn't know that